extensions/azdata/src/typings/azdata-ext.d.ts

```diff
@@ -185,7 +185,7 @@ declare module 'azdata-ext' {
 		},
 		spec: {
 			engine: {
-				extensions: {
+				extensions?: {
 					name: string // "citus"
 				}[],
 				settings: {
```

Comment:
It looks like settings is also optional - I'd suggest changing that too and making sure we're handling that correctly